pom.xml

```diff
@@ -387,5 +387,10 @@
       <artifactId>wdtk-dumpfiles</artifactId>
       <version>0.10.0</version>
     </dependency>
+    <dependency>
```

Comment:
Added this for creating mocks in the unit tests, hopefully this should be useful for later development - I'm surprised we didn't use mocks anywhere already

Reply:
yes, we've not been particularly rigorous in testing... this is my fault in not cracking the whip hard enough :)